app.test.js

```diff
@@ -3,6 +3,7 @@ const request = require('supertest');
 const connection = require('./db/connection')
 const seed = require("./db/seeds/seed")
 const testData = require("./db/data/test-data")
+const data = require('./endpoints.json')
```

Comment:
Can you think of a better variable name than data for this? Data is not a very descriptive variable name and could mean anything, but we know what this variable is and what it contains.